### PR TITLE
Fix #130: Wrong answer for RPC server method getaccountstate

### LINKED ISSUE #130: [fixed by #131] Wrong answer for RPC method getaccountstate
https://docs.neo.org/en-us/node/cli/2.9.4/api/getaccountstate.html

our response is
```json
{
    "jsonrpc": "2.0",
    "result": {
        "version": 0,
        "address": "AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y",
        "script_hash": "0xe9eed8dc39332032dc22e5d6e86332c50327ba23",
        "frozen": false,
        "votes": [],
        "balances": {
            "602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7": "72099.99960000",
            "c56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b": "99989900"
        }
    },
    "id": 1
}
```

in documentation:
```json
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "version": 0,
        "script_hash": "0x1179716da2e9523d153a35fb3ad10c561b1e5b1a",
        "frozen": false,
        "votes": [],
        "balances": [{
            "asset": "0xc56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b",
            "value": "94"
        }]
    }
}
```

1. In our response `Balance` is json.Object (map[string]string), but in documentation it's slice of objects
2. Address not need in the answer

I'll fix it in a few days

### PR DESCRIPTION
**Pull request by im-kulikov**
- fixed RPC Server response
- fixed RPC Server tests
- remove unused package from go.mod

### CODE REVIEW

**Comment by dauTT** [comment]
Good catch! :+1: 

It seems that the neo-python implementation is not update. I will raise an issue there. Thanks.

**Comment by im-kulikov** [comment]
@dauTT 
- [2.8.x](https://docs.neo.org/en-us/node/cli/2.8.0/api/getaccountstate.html) documentation says the same thing
- [2.7.x](https://docs.neo.org/en-us/node/cli/2.7.6/api/getaccountstate.html) too

**Comment by dauTT** [comment]
:+1:  the neo-python version that I am using is a little bit outdated. It seems that meanwhile,  they have partially fixed the issue. But, they still provide the `address`  information. :smile:

**Comment by im-kulikov** [comment]
yeah.. I'm not sure why they're adding it

File: pkg/rpc/params.go [hunk 1]
```diff
@@ -60,3 +61,20 @@ func (p Params) ValueAtAndType(index int, valueType string) (*Param, bool) {
 
 	return nil, false
 }
+
+func (p Params) Value(index int) (*Param, *Error) {
+	if len(p) <= index {
+		return nil, newError(-2146233086, http.StatusOK, "Index was out of range. Must be non-negative and less than the size of the collection.\nParameter name: index", "", nil)
```
Comment (aprasolova) [inline pkg/rpc/params.go]: I propose to put this magic error code (and below one) into the constant with mnemonic name (disregarding the fact it is used only once). Then, code would be slightly readable.

**Comment by hal0x2328** [comment]
Award im-kulikov 200 points
Award dauTT 50 points for review
Award @aprasolova 50 points for review

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/144dc06fbc54dd84165af08ddd9c01e8a720ca1bb05471dfd1af1503160f245f

**Pull request merged**